avaframe/in3Utils/fileHandlerUtils.py

```diff
@@ -204,7 +204,7 @@ def getTimeIndex(cfg, Fields):
     return tStepsOk
 
 
-def getDFAData(avaDir, workDir, suffix, comModule='com1DFA', nameDir=''):
+def getDFAData(avaDir, workDir, suffix, comModule='com1DFA', nameDir='', countStart=0):
```

Comment:
Same question as before, what is the countStart needed for?

Reply:
see answer from above

Reply:
removed